x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list_statuses.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiFlexItem, EuiHealth } from '@elastic/eui';
+import { getIsExperimentalFeatureEnabled } from '../../../../common/get_experimental_features';
+
+import {
+  RULE_STATUS_ACTIVE,
+  RULE_STATUS_ERROR,
+  RULE_STATUS_WARNING,
+  RULE_STATUS_OK,
+  RULE_STATUS_PENDING,
+  RULE_STATUS_UNKNOWN,
+  RULE_LAST_RUN_OUTCOME_SUCCEEDED_DESCRIPTION,
+  RULE_LAST_RUN_OUTCOME_WARNING_DESCRIPTION,
+  RULE_LAST_RUN_OUTCOME_FAILED_DESCRIPTION,
+} from '../translations';
+
+interface RulesListStatusesProps {
+  rulesStatuses: Record<string, number>;
+  rulesLastRunOutcomes: Record<string, number>;
+}
+
+export const RulesListStatuses = (props: RulesListStatusesProps) => {
+  const { rulesStatuses, rulesLastRunOutcomes } = props;
+
+  const isRuleLastRunOutcomeEnabled = getIsExperimentalFeatureEnabled('ruleLastRunOutcome');
+
+  if (isRuleLastRunOutcomeEnabled) {
+    return (
+      <>
```

Comment:
I am wondering if we just should bring the  <EuiFlexGroup alignItems="center" gutterSize="none"> inside of this component